src/vm/amd64/JitHelpers_InlineGetThread.asm

```diff
@@ -147,6 +147,11 @@ align 16
         jmp     JIT_Box
 NESTED_END JIT_BoxFastMP_InlineGetThread, _TEXT
 
+RELATIVE_PTR_LOAD macro Reg, BaseReg, Offset
```

Comment:
Historic context: TemplateMTs used to be shared between different ArrayTypeDescs in the past. It is not the case anymore, except for array of pointers and generics (https://github.com/dotnet/coreclr/blob/b258792e59b09060f54e0c9bbd31edc3e67d1ae8/src/vm/typedesc.cpp#L216). It should not be a problem in this case. There is 1:1 mapping between ArrayTypeDesc and TemplateMT for all practical purposes.